extensions/healthie/actions/__tests__/errors.ts

```diff
@@ -0,0 +1,67 @@
+import { responseMiddleware } from '../../graphqlClient'
+
+const response = {
+  // Type is a bit more complex, but `responseMiddleware` doesn't use that field
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+  headers: {} as Headers,
+  status: 200,
+  data: {
+    createTask: {
+      task: null,
+      messages: [
+        {
+          field: 'base',
+          message: 'You do not have permission to create this task.',
+        },
+      ],
+    },
+  },
+}
+
+describe('responseMiddleware', () => {
+  const responseMiddlewareMock = jest.fn(responseMiddleware)
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('Should throw error when containing messages', async () => {
+    expect(() => {
+      responseMiddlewareMock(response)
+    }).toThrowError({
+      name: 'HealthieError',
+      message:
+        'Error in Healthie: invalid object name or field, or a record is not found.',
```

Comment:
@michal-grzelak Is it possible to grab the actual error message? In this case, I'd expect we throw an error with the following messag: `You do not have permission to create this task.`